src/test/scripts/functions/binary/tensor/ElementwiseAdditionTest.dml

```diff
@@ -21,4 +21,4 @@ d2 = matrix($3, 1, $4);
 A = tensor($5, d1);
 B = tensor($6, d2);
 C = A + B;
-print(toString(C));
+# print(toString(C));
```

Comment:
Why not delete this line entirely?

Reply:
agree, i did leave multiple of these where i out comment the to string functions.